client/my-sites/pages/page-list.jsx

```diff
@@ -220,6 +181,43 @@ class Pages extends Component {
 		);
 	}
 
+	showBlogPostsPage() {
+		const { site, homepageType, isFSEActive } = this.props;
+		const { search, status } = this.props.query;
+
+		return (
+			( ! config.isEnabled( 'unified-pages/virtual-home-page' ) ||
+				/** Blog posts page is for themes that don't support FSE */
+				! isFSEActive ) &&
+			site &&
+			homepageType === 'posts' &&
+			/** Under the "Published" tab */
+			status === 'publish,private' &&
+			/** Without any search term */
+			! search
+		);
+	}
+
+	/**
+	 * Show the virtual homepage
```

Comment:
Nit: This comment does not add any information (it's exactly the same as the method name)